src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -495,6 +518,13 @@ public class Options
                 HelpText = "Data file has header with feature names. Header is read only if options 'hs' and 'hf' are not specified.")]
             public bool HasHeader;
 
+            /// <summary>
+            /// Whether the members name of the class passed as type argument should be mapped to <see cref="LoadColumnNameAttribute"/>.
+            /// When <see langword="true"/> requires the <see cref="HasHeader"/> option to be <see langword="true"/>.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Use class properties as LoadColumnNameAttribute to read from text file", ShortName = "automap")]
+            public bool AutoMapLoadColumns;
```

Comment:
>AutoMapLoadColumns [](start = 24, length = 18)

This is only used in the generic overload, that creates a `TextLoader` from a given type, and not in the overload that takes an `Options` object, so it doesn't need to be a part of the `Options` class.